src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagEntryValidator.cs

```diff
@@ -171,7 +171,7 @@ private static void ValidatePrivateCreator(DicomTag tag, string privateCreator,
 
             try
             {
-                DicomElementMinimumValidation.ValidateLO(privateCreator, nameof(privateCreator));
+                DicomLongStringValidation.Validate(privateCreator, nameof(privateCreator));
```

Comment:
Was'nt the previous way of calling better? What needs to be validated for a VR type was a concern in one API, now it is callers responsibility to figure this out.

Reply:
If I understand correctly, what you mean is previously caller only need to know `DicomElementMinimumValidation`, and now need to figure out which class (it's `DicomLongStringValidation`) in this case for validation?

yeah, I think that's the Cons of this approach. 

In the other side, I feel this approach make `DicomElementMinimumValidator` more clean and easier to extended. 
I feel `DicomElementMinimumValidator` is the "main scenario" while  validating LongString in `ExtendedQueryTagEntryValidator` is kind of "edge case", so  I feel this way is better.